DSharpPlus.Commands/Processors/TextCommands/Parsing/DefaultPrefixResolver.cs

```diff
@@ -8,16 +9,31 @@ namespace DSharpPlus.Commands.Processors.TextCommands.Parsing;
 
 public sealed class DefaultPrefixResolver
 {
-    public string Prefix { get; init; }
+    /// <summary>
+    /// Prefixes which will trigger command execution
+    /// </summary>
+    public string[] Prefixes { get; init; }
+    
+    /// <summary>
+    /// Setting if a mention will trigger command execution
+    /// </summary>
+    public bool AllowMention { get; init; }
 
-    public DefaultPrefixResolver(string prefix)
+    /// <summary>
+    /// Default prefix resolver
+    /// </summary>
+    /// <param name="allowMention">Set wether mentioning the bot will count as a prefix</param>
+    /// <param name="prefix">A list of prefixes which will trigger commands</param>
+    /// <exception cref="ArgumentException">Is thrown when no prefix is provided or any prefix is null or whitespace only</exception>
+    public DefaultPrefixResolver(bool allowMention, params string[] prefix)

```

Comment:
the new parameter should be mentioned in docs/commands/introduction.md

Reply:
will add

Reply:
please spellcheck the new commit :when: